### PR TITLE
[version] Fix --version to print tags for swift.

### PR DESCRIPTION
**Pull request by dcci**
<rdar://problem/47154462>

### CODE REVIEW

**Comment by dcci** [comment]
I need to update a test

**Comment by dcci** [comment]
@swift-ci please test and merge

File: packages/Python/lldbsuite/test/help/TestHelp.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def test_help_version(self):
         version_str = self.version_number_string()
         match = re.match('[0-9]+', version_str)
         search_regexp = ['lldb( version|-' + (version_str if match else '[0-9]+') + '| \(swift-.*\)).*\n']
-        search_regexp[0] += '  Swift-\d+\.\d+'
+        search_regexp[0] += 'Swift version.*LLVM.*Clang.*Swift'
```
Comment (jrose-apple) [inline packages/Python/lldbsuite/test/help/TestHelp.py:95]: This regexp only works on builds from source, not tagged builds.

Comment (dcci) [reply packages/Python/lldbsuite/test/help/TestHelp.py:95]: I assume I can just relax this test, it doesn't matter that much.

**Pull request merged**